cli/src/torii/routing.rs

```diff
@@ -344,6 +344,22 @@ mod subscription {
     }
 }
 
+#[iroha_futures::telemetry_future]
+#[cfg(feature = "telemetry")]
+async fn handle_version<W: WorldTrait>(wsv: Arc<WorldStateView<W>>) -> Json {
+    use iroha_version::Version;
+
+    #[allow(clippy::expect_used)]
+    reply::json(
+        &wsv.blocks()
```

Comment:
Bad idea. What if we changed the version after running Iroha for a while. `blocks().last()` on the other hand should contain either `genesis` since it's the first **and** last block, while it also shows the latest supported API version.

Reply:
Fair enough, fixed that.